terraform/aks/config/staging.tfvars.json

```diff
@@ -3,6 +3,7 @@
     "namespace": "git-test",
     "environment": "staging",
     "infra_key_vault_name": "s189t01-gse-stg-inf-kv",
+    "key_vault_name": "s189t01-gse-stg-app-kv",
```

Comment:
I believe key_vault_name can be removed from all the tfvars files

Reply:
needed by review tfvars though. others removed